.github/workflows/site.yml

```diff
@@ -62,8 +62,9 @@ jobs:
       - name: Deploy Production Site
         working-directory: ./packages/site
         if: github.ref == 'refs/heads/main'
-        run: node scripts/deploy/aws.js
+        run: yarn run deploy:prod
```

Comment:
Let's just call `yarn run formideploy deploy --production` (and the likes) here, since we'd never run this manually, I suppose

Reply:
Good psot, done